docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -177,6 +178,7 @@ const DocsPageWrapper = props => {
 
   return (
     <BpkContentContainer className={getClassName('bpkdocs-page-wrapper')}>
+      <Helmet title={props.title} />
```

Comment:
Previously, `DocsPageBuilder` set `window.title`. Because of this, pages migrated to the new format had the window title `Backpack | Backpack`, instead of the component. By moving this to `DocsPageWrapper`, component pages will now get the correct window title (e.g. `Toast | Backpack`).